modules/default.nix

```diff
@@ -8,6 +8,7 @@ with lib;
 let
 
   modules = [
+    ./envvars.nix
```

Comment:
I think it would be good to make it clear that this is about environment variables set by the shell, something like `shell-env-vars.nix` or `shell-session-vars.nix`.